### PR TITLE
[ALS-6655 & ALS-6699] Login aesthetics, Themes, and bug fixes

### PR DESCRIPTION
**Pull request by JamesPeck**

### CODE REVIEW

File: src/routes/(authentication)/login/+layout.svelte [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
-<slot />
+<style>
+  .full-height {
+    height: calc(100% - 56px);
+  }
+  .footer {
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    background-color: #f5f5f5;
```
Comment (srpiatt) [inline src/routes/(authentication)/login/+layout.svelte:33]: Is this color something we can make a variable for at the postcss level? Might be annoying to find it later.

File: src/lib/components/Logo.svelte [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  let clazz = '';
+  export { clazz as class };
+  const finalClass = `colors max-h-20 ${clazz}`;
+  //read setting from .env
+  const useLogoImg = import.meta.env.VITE_LOGO;
```
Comment (srpiatt) [inline src/lib/components/Logo.svelte:6]: Maybe add the branding/theme specific env variables to the example one.

File: src/routes/(authentication)/login/+layout.svelte (hunk 1 above)
Comment (JamesPeck) [reply src/routes/(authentication)/login/+layout.svelte:33]: This class is never used haha

File: src/lib/components/Logo.svelte (hunk 2 above)
Comment (JamesPeck) [reply src/lib/components/Logo.svelte:6]: I will tackle this in another ticket

**Pull request merged**